doc/iterations/2024.5/shapes/configurable_visibility.adoc

```diff
@@ -0,0 +1,47 @@
+= Configurable visibility
+
+== Problem
+
+Sirius Web doesn't provide a way for specifiers to set the default visibility of a node (hidden/visible). 
+It also doesn't provide a way to let specifiers define tools that hide/reveal nodes as part of other operations (e.g. to reveal a hidden node when a child node is created).
+Nodes are always created visible, and the only way to hide them right now is that the end user does it manually.
+
+== Key Result
+
+- Specifiers must be able to define NodeDescriptions that are hidden/visible by default.
```

Comment:
must be able to configure the state of node by default, hidden/faded/visible

Reply:
fixed